app/src/main/java/com/duckduckgo/app/onboarding/ui/page/DefaultBrowserPage.kt

```diff
@@ -99,6 +103,18 @@ class DefaultBrowserPage : OnboardingPageFragment() {
         outState.putBoolean(SAVED_STATE_LAUNCHED_DEFAULT, userTriedToSetDDGAsDefault)
     }
 
+    private fun applyStyle() {
+        activity?.window?.apply {
+            clearFlags(WindowManager.LayoutParams.FLAG_TRANSLUCENT_STATUS)
+            addFlags(WindowManager.LayoutParams.FLAG_DRAWS_SYSTEM_BAR_BACKGROUNDS)
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+                decorView.systemUiVisibility = View.SYSTEM_UI_FLAG_LIGHT_STATUS_BAR
+            }
+            statusBarColor = Color.WHITE
```

Comment:
This is working but I found that we have a similar code in `WelcomePage`.

In there, instead of using `statusBarColor = Color.WHITE`, we have the following:
````
    private fun applyFullScreenFlags() {
        activity?.window?.apply {
            clearFlags(WindowManager.LayoutParams.FLAG_TRANSLUCENT_STATUS)
            addFlags(WindowManager.LayoutParams.FLAG_DRAWS_SYSTEM_BAR_BACKGROUNDS)
            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
                decorView.systemUiVisibility = View.SYSTEM_UI_FLAG_LIGHT_STATUS_BAR
            }
            decorView.systemUiVisibility += View.SYSTEM_UI_FLAG_LAYOUT_FULLSCREEN
            statusBarColor = Color.TRANSPARENT
        }
        ViewCompat.requestApplyInsets(longDescriptionContainer)
    }
```

This is also working for me, do you now the differences? which one is more correct? transparent in case the background changes?

Reply:
Yeah I think transparent makes more sense. We cannot use the `SYSTEM_UI_FLAG_LAYOUT_FULLSCREEN` flag though because it would mess the alignment on the instructions card. This screen cannot be fullscreen because of that.